components/brave_wallet_ui/components/shared/create-placeholder-icon/index.tsx

```diff
@@ -55,8 +55,7 @@ function withPlaceholderIcon (WrappedComponent: React.ComponentType<any>, config
 
     const isValidIcon = React.useMemo(() => {
       if (isRemoteURL || isDataURL) {
-        const url = new URL(asset.logo)
-        return isValidIconExtension(url.pathname)
+        return tokenImageURL?.includes('data:image/') ? true : isValidIconExtension(new URL(asset.logo).pathname)
```

Comment:
why is this `includes` instead of startsWith?

Reply:
@diracdeltas  Because token image urls start with `chrome://erc-token-images`

Reply:
Shouldn't be `chrome-untrusted://erc-token-images` here?

Reply:
@thypon No, only NFT images are using chrome-untrusted, all existing icons use `chrome://erc-token-images`.